.github/workflows/lint_and_test.yaml

```diff
@@ -111,3 +111,10 @@ jobs:
         run: go install golang.org/x/tools/cmd/goimports@latest
       - name: Lint
         run: make check
+
+  codespell: # call reusable workflow from central '.github' repo
+    uses: open-component-model/.github/.github/workflows/codespell.yml@main
+    secrets: inherit
+    with:
+      codespell-ignore: .github/config/wordlist.txt
+      codespell-files-glob: ./**/*.y*ml ./**/*.go
```

Comment:
the globbing for `y*ml` looks weird. I guess it shall cover `yaml` and `yml`?

Reply:
yes

Reply:
actually these are the default yalues (see https://github.com/open-component-model/.github/blob/main/.github/workflows/codespell.yml). Do we have to set it again if we use reusable workflows?

Reply:
No, you don't need to set it again. On the other hand, when you're copy/pasting this reusable snipped into other repos, it's easier to understand, what can be adjusted. You would not have to go into the original repo and check what's possible. But if you prefer to not have it here we can of course remove it. Basically I don't like relying on defaults. They are good for beginners to have an easy jumpstart, but as soon as someone changes the default, things become complicated.

Reply:
yes, i feel the same